helpers/migrate.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python3
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import subprocess
+import sys
+import shutil
+import re
+
+
+class GSuiteMigration:
+    """
+    Migrate the resources from a flat project factory to match the new
+    module structure created by the G Suite refactor.
+    """
+
+    CORE_PROJECT_FACTORY_SELECTORS = [
+        ("google_organization", "org"),
+        ("google_compute_default_service_account", "default"),
+        ("null_data_source", "data_final_group_email"),
+        ("null_data_source", "data_group_email_format"),
+        ("google_compute_shared_vpc_service_project", None),
+        ("google_compute_subnetwork_iam_member", None),
+        ("google_project", None),
+        ("google_project_iam_member", None),
+        ("google_project_service", None),
+        ("google_project_usage_export_bucket", None),
+        ("google_service_account", None),
+        ("google_service_account_iam_member", None),
```

Comment:
`google_service_account_iam_member.service_account_grant_to_group` remains in `gsuite_enabled` so this rule must be restricted.

In case we test this script on test state, `google_service_account_iam_member.additive_service_account_grant_to_group` remains in `full` fixture.